config/__init__.py

```diff
@@ -17,7 +17,7 @@ class Config:
         'SLACK_API_TOKEN': 'slack_api_token',
         'SLACK_NOTIFICATION_CHANNEL': 'slack_notification_channel',
         'SLACK_ANNOUNCEMENT_CHANNEL': 'slack_announcement_channel',
-
+        'SLACK_DONUT_CHANNEL': 'slack_donut_channel',

```

Comment:
```suggestion
        'SLACK_PAIRINGS_CHANNEL': 'slack_pairings_channel',

```

etc - not that the chances of being sued is very high, but we should probably stick on the safe side with a more generic name like "pairings" for this feature. Also, not everyone is familiar with donut either

Reply:
(or `matchings`)